README.md

```diff
@@ -1,5 +1,5 @@
-# cc19
```

Comment:
I will ask Prof. Robbins to check this whether you should be updating the README

Reply:
Per comment below, please remove the README from the pull request.

Reply:
How do I remove a file from a pull request? Do I need to submit another PR?

Reply:
@jtr13 Can you advise?

Reply:
Go back to your local branch `cc_among_us_project` and delete the README file. Then commit and push again and the PR will be automatically updated.